config/environments/test.rb

```diff
@@ -36,5 +36,5 @@
   config.active_support.deprecation = :stderr
 
   # Raises error for missing translations
-  # config.action_view.raise_on_missing_translations = true
+  config.action_view.raise_on_missing_translations = true
```

Comment:
Is this something that we would have to disable on apps that doesnt use I18n in the future?
if so does it need to be on by default?